### DISCUSSION TITLE
Difference between click and drag on node handles

### DISCUSSION THREAD

**Discussion by CamboimGabriel** [xyflow/xyflow#2195] [Q&A]
Hey, I'm trying to create a clickable handle but it creates some bugs with the already implemented dragging function of handles, is there a way I can create a 5px buffer or so to only activate the dragging after the mouse movement pass this buffer? (without change the library)

**Comment by ArturMakly** [comment]
try quickly clicking this green circle:
<img width="379" alt="image" src="https://user-images.githubusercontent.com/40069318/171753567-e204a6ca-1c93-4b41-a623-d0c03c9a016d.png">

1. [go here](https://userflows.d.visualsitemaps.com/)
2. click a Shape to add
3. roll over and click the green Connector
4. do this a few time really fast.. you will see that the click is too precise and if you move 1-2px.. it will think it is being dragged.
it needs some 'wiggle' room!
